policyengine_us/parameters/gov/states/mt/tax/income/subtractions/social_security/exceeding_income_cap.yaml

```diff
@@ -0,0 +1,22 @@
+description: Montana subtracts this amount from exceeding part of modified income (compared with income cap) depending on filing status. 
+metadata:
+  label: Montana social security subtraction exceeding income cap
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2022 Montana Individual Income Tax Return Form, Taxable Social Security Benefits Schedule, Line 12
+    href: https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf
```

Comment:
specify the exact page for the title and link, for example:
- title: 2022 Montana Individual Income Tax Return Form, page6, Taxable Social Security Benefits Schedule, Line 12
    href: https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf#page=6